### ISSUE TITLE
HuBoard, Waffle, GitHub Projects, CRM boards

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/standard-development-handbook#6]
~~[`sample-data`](https://huboard.com/open-contracting/sample-data) and [`standard`](https://huboard.com/open-contracting/standard) have HuBoard integrations.~~ [Deleted]

[`ocds-show`](https://github.com/open-contracting/ocds-show/projects) and the [organization](https://github.com/orgs/open-contracting/projects) have GitHub Projects.

~~`standard` has a webhook for Waffle, but it seems to be failing.~~ [Deleted]

We also have a [board](http://crm.open-contracting.org/projects/tech/agile/board) in the CRM.

* [x] Choose between ~~HuBoard, Waffle,~~ GitHub Projects, CRM boards
* [x] Delete the webhook/project/board of the options we don't prefer
* [ ] ~~If we didn't choose GitHub Projects, [uncheck "Enable projects for all repositories"](https://github.com/organizations/open-contracting/settings/projects). Otherwise, create issues for the issue-less cards in our two projects.~~
* [x] If we didn't choose HuBoard, delete the numeric labels it created
* [x] Document how we use whichever option(s) we prefer

**Comment by jpmckinney** [comment]
I just noticed that GitHub Projects allows adding cards that are not attached to issues…. This isn't good, as people expect to have visibility of a repo's issues within the issues tab. If we're using GitHub Projects, we should create issues and then add issue-cards to the project board.

**Comment by jpmckinney** [comment]
Having reviewed the entire CRM board, we should definitely keep that (its uses and features are somewhat distinct and complementary to what GitHub offers).

I think we only need one of HuBoard or GitHub Projects. I can see their utility in organizing GitHub issues in Kanban style (as long as we don't create issueless notes!). However, we only need one.

I think we can delete Waffle.

**Comment by Bjwebb** [comment]
Yes, pretty sure Waffle cran go.

For software development we've been moving from Huboard to the (newer) GitHub Projects. I think Huboard hasn't been used for schema development for a while either, but worth double checking that with @timgdavies

**Comment by jpmckinney** [comment]
In sample-data, since only https://github.com/open-contracting/sample-data/issues/10 is 'ready' and everything else is 'done' or 'backlog', I'm deleting the labels HuBoard created.

**Comment by timgdavies** [comment]
Correct. We used HuBoard during last round of schema issue prioritisation as the interface was easier to use with external partners - but I think for any future rounds, we would want to use GitHub projects now that it is more mature. Happy for HuBoard to be entirely retired and tags removed. 

For the CRM board: this was created to represent the full range of technical assets that need attention at various points, and so is less project management, more about getting a full view of things that will need attention *at some point*, even if not actively so now.

**Comment by jpmckinney** [comment]
I removed HuBoard's labels and webhooks on `standard`. I reviewed the tagged issues and it didn't look like any information was lost by removing the labels (the issues still report the history of which labels were added/removed).

Last step is to document how/when we use GitHub Projects. (We don't need to explain how Kanban works - we can just link 'Kanban' to Wikipedia or similar.) I find that using cards that don't link to issues makes those cards somewhat invisible to people who are accustomed to only look at the Issues tab, so we may want to decide how/when we use such cards.
